bin/index.js

```diff
@@ -10,22 +10,55 @@ const {
   generatePdfFromBuildWithConfig,
 } = require("../lib");
 
+function generatePdfOptions({ sandbox, margin, format, printBackground }) {
+  return {
+    puppeteerArgs: sandbox ? [] : ["--no-sandbox"],
+    puppeteerPdfOpts: { margin, format, printBackground },
+  };
+}
+
+function parseMargin(marginString) {
+  const matches = [...marginString.match(/\d+\w{0,2}/g)];
+  if (matches.length !== 4) {
+    throw Error(
+      `Was expecting exactly 4 margin specifiers, instead got ${matches.length}. Margin specifier was "${marginString}".`
+    );
+  }
+  const [top, right, bottom, left] = matches;
+  return { top, right, bottom, left };
+}
+
 program
   .version(require("../package.json").version)
   .name("docusaurus-pdf")
   .usage("<initialDocsUrl> [filename]")
-  .description("Generate PDF from initial docs url")
-  .arguments("<initialDocsUrl> [filename]")
-  .action((initialDocsUrl, filename) => {
-    generatePdf(initialDocsUrl, filename)
-      .then(() => {
-        console.log(chalk.green("Finish generating PDF!"));
-        process.exit(0);
-      })
-      .catch((err) => {
-        console.error(chalk.red(err.stack));
-        process.exit(1);
-      });
```

Comment:
I like the idea to move the action to a separate command named `from-website`. 
When doing so I think we should in addition remove `.usage("<initialDocsUrl> [filename]")` at line 34 of the new version of `bin/index.js` because there is no action anymore which does anything with those parameters.

This change would probably break our current API if I don't miss anything. This would lead to a new major release. 

Otherwise we could keep this action as default behavior for `docusaurus-pdf` and introduce `docusaurus-pdf from-website` as an alias.

Reply:
I've set:

```js
program
  .command("from-website <initialDocsUrl> [filename]", {
    isDefault: true, // this means it is the default action (e.g. when no command is specified)
```

Reply:
Great, I obviously missed that line and did something wrong trying to use the default command 👍